apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -72,9 +71,6 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
 
             pluginFinder = new PluginFinder(new PluginBootstrap().loadPlugins());
 
-        } catch (ConfigNotFoundException ce) {
```

Comment:
We need to catch this. In the `no config` scenario, the app should startup successfully.

Reply:
> We need to catch this. In the `no config` scenario, the app should startup successfully.

It will be caught by `[HERE](https://github.com/apache/skywalking/blob/4df7129ac15308582c4a77185f2df19cf619c074/apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/SnifferConfigInitializer.java#L68)`

```diff
    public static void initialize(String agentOptions) {

        try (InputStreamReader configFileStream = loadConfig()) {
            Properties properties = new Properties();
            properties.load(configFileStream);
            properties.forEach((key, value) -> {
                if (value instanceof String) {
                    properties.replace(key, PropertyPlaceholderHelper.INSTANCE.replacePlaceholders((String)value, properties));
                }
            });
            ConfigInitializer.initialize(properties, Config.class);
+        } catch (Exception e) {
            logger.error(e, "Failed to read the config file, skywalking is going to run in default config.");
        }
```

Reply:
Got it, make sense to me. Ignore this comment.